typing/layouts.ml

```diff
@@ -491,10 +548,12 @@ end
     open Format
 
     let internal ppf : internal -> unit = function
-      | Any         -> fprintf ppf "Any"
-      | Sort s      -> fprintf ppf "Sort %a" Sort.Debug_printers.t s
+      | Any { missing_cmi_for } ->
+          fprintf ppf "Any { missing_cmi_for = %a }"
+            (Misc.Stdlib.Option.print Path.print) missing_cmi_for
```

Comment:
A problem I have a lot is the typed tree printer is so verbose it can be hard to find the relevant bits.  What do you think about only printing this record if it's Some?  (I do not feel strongly either way)